tools/osv-scanner/cmd/osv-scanner/config.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+)
+
+type LoadConfigErrorType int64
+
+const (
+	GlobalConfigSet LoadConfigErrorType = iota
```

Comment:
these all seem to only be internally used. Can we either:

- make these private, or 
- just use plain strings directly via `errors.New`/`fmt.Errorf` instead? It seems like this distinction is not needed programatically anywhere anyway, so we can keep this simple.